### PR TITLE
Clarify radio dongle flashing instructions

### PR DESCRIPTION
**Pull request by jonathanlew**
## Please fill out the following before requesting review on this PR
There was some trouble flashing radio dongles using the existing instructions. This PR attempts to clarify a few issues.
### Testing Done
manually on dongles

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [ ] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -211,14 +212,22 @@ bazel run -c dbg --run_under="valgrind --tool=callgrind --callgrind-out-file=/tm
...
 7. To see print outs from the robot, run `sudo cat /dev/ttyACM0`. If `ttyACM0` isn't the right device, run `ls /dev/tty*` with the robot disconnected, and again with the robot connected, and replace `ttyACM0` with the new device that has been added. The radio dongle does not have this feature.
 
+## Flashing the Radio Dongle
+1. Make sure you've followed [Installing Firmware Dependencies](#installing-firmware-dependencies), and have a dongle plugged into your computer. Do not plug both the dongle and the robot at the same time!
+2. From the `src` folder, to flash the radio dongle, run `bazel run --cpu=stm32f4 //firmware/tools:legacy_flash_firmware radio_dongle`.
+3. If the dongle has a blue pcb, then follow step (a). If the dongle has a green pcb then follow step (b).
+    (a) Hold down the red button while unplugging and plugging in the dongle. There should be a single orange light on. Then run the script described in step 2.
+    (b) Hold down the yellow button while running the script described in step 2.
+4. There should be a progress bar indicating the flashing progress, hold the switch back until the process finishes.
```
Comment (akhilveeraghanta) [inline docs/getting-started.md]: nit: dongle has a different switch, back -> down

**Review by akhilveeraghanta — COMMENTED** [review]
just a nit

File: docs/getting-started.md (hunk 1 above)
Comment (jonathanlew) [reply docs/getting-started.md]: fixed

**Review by akhilveeraghanta — DISMISSED** [review]

File: docs/getting-started.md [hunk 2]
```diff
@@ -211,14 +212,21 @@ bazel run -c dbg --run_under="valgrind --tool=callgrind --callgrind-out-file=/tm
...
 7. To see print outs from the robot, run `sudo cat /dev/ttyACM0`. If `ttyACM0` isn't the right device, run `ls /dev/tty*` with the robot disconnected, and again with the robot connected, and replace `ttyACM0` with the new device that has been added. The radio dongle does not have this feature.
 
+## Flashing the Radio Dongle
+1. Make sure you've followed [Installing Firmware Dependencies](#installing-firmware-dependencies), and have a dongle plugged into your computer. Do not plug both the dongle and the robot at the same time!
+2. From the `src` folder, to flash the radio dongle, run `bazel run --cpu=stm32f4 //firmware/tools:legacy_flash_firmware radio_dongle`.
+3. If the dongle has a blue pcb, then follow step (a). If the dongle has a green pcb then follow step (b). There should be a progress bar indicating the flashing progress.
+    (a) Hold down the red button while unplugging and plugging in the dongle. There should be a single orange light on. Then run the script described in step 2.
+    (b) Hold down the yellow button while running the script described in step 2.
```
Comment (Resocram) [inline docs/getting-started.md:230]: image

This didn't render well on my screen

File: docs/getting-started.md [hunk 3]
```diff
@@ -211,14 +212,21 @@ bazel run -c dbg --run_under="valgrind --tool=callgrind --callgrind-out-file=/tm
...
+3. From the `src` folder, to flash the radio dongle, run `bazel run --cpu=stm32f4 //firmware/tools:legacy_flash_firmware robot`
+4. For the robot, make sure the robot is in bootloader mode (BL switch on the piano keys is down), and push the power switch away from the dribbler (i.e towards the back of the robot) and hold it there before running the command.
 5. There should be a progress bar indicating the flashing progress, hold the switch back until the process finishes.
 6. When the process is finished, release the power switch, push the BL switch back up, and the robot now has been flashed!
 7. To see print outs from the robot, run `sudo cat /dev/ttyACM0`. If `ttyACM0` isn't the right device, run `ls /dev/tty*` with the robot disconnected, and again with the robot connected, and replace `ttyACM0` with the new device that has been added. The radio dongle does not have this feature.
 
+## Flashing the Radio Dongle
+1. Make sure you've followed [Installing Firmware Dependencies](#installing-firmware-dependencies), and have a dongle plugged into your computer. Do not plug both the dongle and the robot at the same time!
```
Comment (Resocram) [inline docs/getting-started.md]: Minor comment, the comma is not needed after `Make sure you've followed [Installing Firmware Dependencies](#installing-firmware-dependencies)`

**Review by Resocram — CHANGES_REQUESTED** [review]

File: docs/getting-started.md (hunk 3 above)
Comment (jonathanlew) [reply docs/getting-started.md]: fixed

File: docs/getting-started.md (hunk 2 above)
Comment (jonathanlew) [reply docs/getting-started.md:230]: good catch fixed

**Review by akhilveeraghanta — APPROVED** [review]

**Review by Resocram — APPROVED** [review]
lgtm!

**Pull request merged**